frontend/public/module/k8s/pods.ts

```diff
@@ -248,7 +248,7 @@ export const podPhase = (pod: PodKind): PodPhase => {
         : `Init:ExitCode:${terminated.exitCode}`;
     } else if (waiting && waiting.reason && waiting.reason !== 'PodInitializing') {
       phase = `Init:${waiting.reason}`;
-    } else {
+    } else if (!running && !ready) {
```

Comment:
FYI: After going through the docs linked above, I am still not sure how these conditions are derived, but adding this logic fixes the issue. It would be good to double-check this.